tools/explorer/app.py

```diff
@@ -11,14 +11,133 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Main module for DBO explorer."""
+import ast
+import pyfiglet
 import sys
+from termcolor import colored
+from typing import List
 
-from lib import arg_parser
 from lib import explorer
 from lib import model
-import pyfiglet
+from lib.model import StandardField
+from lib import arg_parser
+
+def _InputFieldsManually(ontology) -> List[StandardField]:
+  """Method to take individual field inputs from user and return as a list.
+
+  Args:
+    ontology: An instance of OntologyWrapper class.
+
+  Returns:
+    A list of StandardField objects.
+  """
+  standard_field_list = []
+  field_number = input('Number of entity fields: ')
+  for i in range(1, int(field_number)+1):
+    field_name = input(f'{i}. What is the field name?\nname: ')
```

Comment:
can you please move this inside to look like: What is the field name #i?

Reply:
Done.